test/unit/spec/common/jobCommChannel-spec.js

```diff
@@ -20,9 +21,11 @@ define([
     const JOB_ID = jcm.PARAMS.JOB_ID,
         JOB_ID_LIST = jcm.PARAMS.JOB_ID_LIST,
         BATCH_ID = jcm.PARAMS.BATCH_ID,
-        BACKEND_JOB_ID = 'job_id',
-        BACKEND_JOB_ID_LIST = 'job_id_list',
-        BACKEND_BATCH_ID = 'batch_id';
+        BACKEND_JOB_ID = jcm.BE_PARAMS.JOB_ID,
+        BACKEND_JOB_ID_LIST = jcm.BE_PARAMS.JOB_ID_LIST,
+        BACKEND_BATCH_ID = jcm.BE_PARAMS.BATCH_ID,
+        JOB_CHANNEL = jcm.CHANNELS.JOB,
+        CELL_CHANNEL = jcm.CHANNELS.CELL;
```

Comment:
Switching out `cell: ...` and `jobId: ...` in the channel data is responsible for the majority of the changes in this PR.